.github/ISSUE_TEMPLATES/1-bug-report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: Report a bug with the WordPress Props Bot.
+labels: ['[Type] Bug']
+assignees:
+  - desrosj
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to improve the overall contributor experience by filling out this bug report!
+  - type: textarea
+    attributes:
+      label: Description
+      description: Please write a brief description of the bug, including what you expect to happen versus what is currently happening.
+      placeholder: |
+        Feature '...' is not working properly. I expect '...' to happen, but '...' happens instead
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Step-by-step reproduction instructions
+      description: Please write the steps needed to reproduce the bug.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+    validations:
+      required: true
```

Comment:
might be able to collapse this into the question above as they're directly related and reduces a required field by 1 for a reporter